docs: describe `env` object for nuxt plugins (#22963)
nuxt/nuxt: docs/2.guide/2.directory-structure/1.plugins.md

})
```

> #### defining env

Reviewer: ```suggestion
#### Defining Plugin Environments
```

> You can use the `env` property to define in which environment your plugin will be ran.

Reviewer: ```suggestion
You can use the `env` property to define the environments in which your plugin will run.
```

> #### defining env
> 
> You can use the `env` property to define the environments in which your plugin will run.

Reviewer: ```suggestion
You can also use the `env` property to define the environments in which your plugin will run.
```

- **islands**
  - **default**: `true`

>   - **description**: You can set this value to `false` if you don't want the plugin to be ran for any server-only or island components.

Reviewer: ```suggestion
  - **description**: You can set this value to `false` if you don't want the plugin to run when rendering server-only or island components.
```
